core/rendered_connection.ts

```diff
@@ -514,12 +514,7 @@ export class RenderedConnection extends Connection {
       return;
     }
     blockShadow.initSvg();
-    blockShadow.render(false);
-
-    const parentBlock = this.getSourceBlock();
-    if (parentBlock.rendered) {
-      parentBlock.queueRender();
-    }
+    blockShadow.queueRender();
```

Comment:
just double checking, will this also cause the parent block to be queued and that's why we got rid of the code below this?